current_spring2026/submit_update.sh

```
@@ -0,0 +1,26 @@
+#$ -pe omp 8
+
+# =============================================================================
+# Load config (universal params + task list)
+# =============================================================================
+# Load miniconda correctly for this SCC setup
+module load miniconda
+
+conda activate spark-rag
+
+# Change to project root — critical for module imports
+cd /projectnb/sparkgrp/ml-bpl-rag-data-subset/temp/ml-bpl-rag/current_spring2026
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)